### PR TITLE
Add return_warning function to commonserverpython

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18261

## Description
used many warning entries lately (11), figured we should have this available same as return_error

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -1381,6 +1381,44 @@ def return_error(message, error='', outputs=None):
...
+        :return: Error entry object
+        :rtype: ``dict``
+    """
+    LOG(message)
+    if warning:
+        LOG(warning)
+    LOG.print_log()
+
+    demisto.results({
+        'Type': entryTypes['error'],
```
Comment (yaakovi) [inline Scripts/CommonServerPython/CommonServerPython.py]:
```suggestion
        'Type': 11,
```

File: Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -1381,6 +1381,44 @@ def return_error(message, error='', outputs=None):
...
+        :type warning: ``str``
+        :param warning: The raw warning message to log (optional)
+
+        :type outputs: ``dict or None``
+        :param outputs: the outputs that will be returned to playbook/investigation context (optional)
+
+        :type ignore_auto_extract: ``bool``
+        :param ignore_auto_extract: Determines if the war-room entry will be auto enriched. Default is false.
+
+        :return: Error entry object
```
Comment (yaakovi) [inline Scripts/CommonServerPython/CommonServerPython.py]:
```suggestion
        :return: Warning entry object
```

File: Scripts/CommonServerPython/CommonServerPython.py [hunk 3]
```diff
@@ -1381,6 +1381,44 @@ def return_error(message, error='', outputs=None):
     sys.exit(0)
 
 
+def return_warning(message, exit=True, warning='', outputs=None, ignore_auto_extract=False):
```
Comment (yaakovi) [inline Scripts/CommonServerPython/CommonServerPython.py]:
```suggestion
def return_warning(message, exit=False, warning='', outputs=None, ignore_auto_extract=False):
```

**Comment by bakatzir** [comment]
@kirbles19 can you review the docstring?
https://github.com/demisto/content/blob/39a012ddfe24d23fdd27abc9b847fda2941d0488/Scripts/CommonServerPython/CommonServerPython.py
lines 1386-1404 - we generate auto documentation in the script helper from that.

**Comment by kirbles19** [comment]
@bakatzir All done.

**Comment by bakatzir** [comment]
@glicht can you review and if valid to force merge?

**Comment by glicht** [comment]
Unit tests are failing. Are you updated from master?

**Comment by bakatzir** [comment]
Last merge from master was a week ago. updated.

**Pull request merged**